textures/metro_control_panel_001_pegboard_wip.py

```diff
@@ -0,0 +1,85 @@
+"""A collection of stuff related to https://stalburg.net/Body_message#Pegs."""
+from typing import Sequence
+
+from infra.utils import chunks
+from textures.metro_control_panel_001_pegboard import (
+    bools_to_int,
+    bools_to_nor_mask,
+    invert_bools,
+    markers_to_bools,
+    metro_control_panel_001_pegboard,
+    PegGroup,
+    pegs_to_bools,
+)
+
+non_print_vals = {*range(33), *range(127, 160), 160, 173}
+
+
+def is_alpha_num(char: str) -> bool:
+    return any(("0" <= char <= "9", "A" <= char <= "Z", "a" <= char <= "z"))
```

Comment:
I'm not sure, but I _think_ I have seen something like `isalnum` somewhere in the standard library. If it exists, please use it here. Also, this function seems wrong, as `is_alpha_num("-.ä-,-.,-.,9")` would evaluate to `True` as you're using the `any` aggregation.